### ISSUE TITLE
forever if block appears on help display for touching block

### ISSUE THREAD

**Issue by normanbauer** [jmoenig/Snap--Build-Your-Own-Blocks#2205]
The help display for the first two touching blocks contain a forever if block which was in BYOB but not in SNAP. I know this is a very minor detail, but should be corrected in the next version.  Interestingly, the help for color is touching block does not contain the forever if block.

**Comment by brianharvey** [comment]
Yup, good catch, will do.

**Comment by normanbauer** [comment]
Brian,  I am trying to use the touching color sensing command in SNAP. In BYOB, clicking on the color square box in the command caused a pipette to appear and moving it to a color and clicking caused the color to be used in the sensing command.  In SNAP, clicking on the touching color command causes a color spectrum to appear. It would practically impossible to select a color from this to match a color on the stage.  Is there a way to use the pipette (or something else) to select a color from the stage/sprite to be used in the touching color predicate block?

**Comment by brianharvey** [comment]
Although the cursor doesn't change to a pipette (maybe it should), you can click anywhere in the Snap<i>!</i> window to choose the color, not only in the color-picker rectangle.

**Comment by normanbauer** [comment]
You are right, clicking on a color works.  Again your documentation is out of synch( from the touching color help):

The help documentation for touching colors says:
 
Get the eye dropper by clicking in this square
Use the eye dropper to click on the color you want.
Color appears in square. 

No eye dropper is shown, just the spectrum of colors.

I hate to be a pest, but I am a 5 year user of BYOB and am accustom to its eccentricities. New software can be like a marriage without a honeymoon.

**Comment by brianharvey** [comment]
So, you're saying the rest of marriage is unpleasant?  :-)

**Comment by normanbauer** [comment]
Well, it loses some of the romance.  LOL.


-----Original Message-----
